pom.xml

```diff
@@ -18,7 +18,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>software.amazon.qldb</groupId>
     <artifactId>amazon-qldb-driver-java</artifactId>
-    <version>2.2.0</version>
+    <version>2.3.0</version>
```

Comment:
Was there another place this needs to be updated?

I remember the versioning was wrong in a link in the project bundle somewhere.

Also, did we update the user agent string?

Reply:
I don't think so.

Here was the last [PR](https://github.com/awslabs/amazon-qldb-driver-java/pull/47/files) to bump the version to `2.2.0`

For the user agent string, it seems to get the [version](https://github.com/awslabs/amazon-qldb-driver-java/blob/master/src/main/java/software/amazon/qldb/QldbDriverImplBuilder.java#L187) from the POM file.

I searched the whole repo for `2.2.0` and the POM file was the only place that showed up.